README.md

```diff
@@ -1 +1,136 @@
-# art-gallery
\ No newline at end of file
+# art-gallery
+
+[You can visit our site from this link]([https://art-gallery-v1.herokuapp.com/])
+
+## Team Leader
+
+- Rana Obeid.
+
+## Team Members
+
+- Lina Ebeid.
+- Mariam Isa.
+- Muhammad Abdulhadi.
+- Alaa Abu Swaireh.
+
+## Summary
+
+Art gallery is an Arab web application which allows artists to view their arts. so each artist has a profile . Customers can buy arts
+
+## Challenge
+
+The inability of painters to sell their paintings and the difficulty in communicating with customers.
+
+## Solution
+
+Our site provides artists with the ability to sell their paintings at reasonable prices and the ability to communicate with the customer appropriately.
+
+## User Stories
+
+### As an artist
+
+- I can sign up / login and have my own profile
+- I can edit my profile data
+- I can upload my paintings in my profile
+
+### As an user
+
+- I can sign up / login and have an account
+- I can Check artworks for different artists
+- I can Search and filter art works
+- I can buy paintings and add it to my cart
+- I can Checkout from cart
+- I can add a review for the artist
+
+### As an admin
+
+- I can delete a artist/user/painting
+
+## Set up the app locally
+
+First clone this repo: `git clone https://github.com/GSG-G8/art-gallery.git`
+
+then run `npm i` and open new terminal then run `cd client` then `npm i` to install the dependencies for the app.
+
+### Database Setup
+
+In terminal type psql or pgcli if installed. Within psql/pcli enter the following commands:
```

Comment:
We created a `buildDatabase.sql` file which will do all these commands instead, So all these commands aren't really necessary